tests/roles/run_tests/templates/cluster-template-controlplane-kubeadm-config-centos.yaml

```diff
@@ -1,8 +1,10 @@
 # CentOS specific controlplane kubeadm config
 preKubeadmCommands:
   - systemctl restart NetworkManager.service
-  - nmcli connection load /etc/sysconfig/network-scripts/ifcfg-eth0
-  - nmcli connection up filename /etc/sysconfig/network-scripts/ifcfg-eth0
+  - nmcli con load /etc/NetworkManager/system-connections/eth0.nmconnection
+  - nmcli connection up filename /etc/NetworkManager/system-connections/eth0.nmconnection
```

Comment:
Are you sure we should have `filename` here? I can see that it is the same way we used to have it, but I don't find this in the documentation (`nmcli con up --help`). Perhaps it is for some older version? I think it should be `nmcli con up eth0` now.

Reply:
I agree with you. filename was used in here. I removed it now. Changed it to interface name.
Thank you for noticing